deployment/src/kubernetes/README.md

```diff
@@ -9,7 +9,7 @@ Please contact us if you are interested in options not listed below.
 
 Managed Kubernetes:
 
-- [DigitalOcean](/deployment/src/kubernetes/DigitalOcean.md)
+- [DigitalOcean](./DigitalOcean.md)
```

Comment:
Absolut path?

Reply:
In case linked markdown files are in the same directory, relative paths work for both Github and Vuepress.